workers/merkle-tree-worker/src/worker.ts

```diff
@@ -0,0 +1,26 @@
+import { expose } from "comlink";
+import { IMerkleTreeWasm, IMerkleTreeWorker } from "./interface";
+
+let merkleTreeWasm: IMerkleTreeWasm;
+
+export const merkleTreeWorker: IMerkleTreeWorker = {
+    async prepare() {
+        merkleTreeWasm = await import("@anonklub/merkle-tree-wasm");
+    },
+
+    async generateMerkleProof(
+        leaves,
+        leaf,
+        depth
+    ): Promise<Uint8Array> {
+        const proof = await merkleTreeWasm.generate_merkle_proof(
+            leaves,
+            leaf,
+            depth
+        );
+
+        return proof;
```

Comment:
style: redundant declaration of `proof` 

There is no functional pros or cons here, just stay consistent with the style of the rest of project: I usually don't declare the variable I want to return ^^